content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -193,9 +193,11 @@ spec:
           - registry_service_name: istio-ingressgateway.istio-system.svc.cluster.local
             port: 443
 
-  # Change the Istio service `type=LoadBalancer` and add the cloud provider specific annotations. See
+  # Change the Istio service `type=LoadBalancer` add the cloud provider specific annotations. See
```

Comment:
I believe you still want to `add` as the current wording doesn't make sense.

Reply:
```suggestion
  # Change the Istio service `type=LoadBalancer` and add the cloud provider specific annotations. See
```

Reply:
Sorry, I misread the word because of negligence.I changed it back to the original content.